tableau-databricks/base

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Databricks Tableau Connector
+Copyright 2019 Databricks, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. -->
+<resources>
+  <string name="databricks">Databricks</string>
+  <string name="server_prompt">Server Hostname</string>
+  <string name="http_path_prompt">HTTP Path</string>
+
+  <string name="authentication_prompt">Authentication</string>
+  <string name="authentication_label_token">Personal Access Token</string>
+  <string name="authentication_label_user_pass">Username / Password</string>
+
+  <string name="username_prompt">Username</string>
+  <string name="password_prompt">Password</string>
+  <string name="token_prompt">Personal Access Token</string>
+
+  <string name="odbc_extra_config_prompt">Extra ODBC options ('key=value;key2=value2;...')</string>
+
+  <string name="metadata_database_prompt">Database</string>
+  <string name="metadata_table_prompt">Table</string>
+</resources>
```

Comment:
Revert this file

Reply:
why do you want to revert this?

Reply:
I added it accidentally. It serves no purpose.

Reply:
ah right, there are localized files below.